### PR TITLE
Angular: Switch to default to inlineStories=true

### PR DESCRIPTION
**Pull request by tmeasday**
Issue: https://github.com/storybookjs/storybook/issues/18176#issuecomment-1340267173

## What I did

Fix up some loose ends

## How to test

Check angular sandboxes

### CODE REVIEW

File: code/addons/docs/angular/README.md [hunk 1]
```diff
@@ -208,18 +207,14 @@ And for `MDX` you can modify it as an attribute on the `Story` element:
...
-import { addParameters } from '@storybook/angular';
+To do so for all stories, update `.storybook/preview.js`:
 
-addParameters({
-  docs: {
-    inlineStories: true,
-  },
-});
+```js
+export const parameters = { docs: { inlineStories: true } };
```
Comment (shilman) [inline code/addons/docs/angular/README.md:217]:
```suggestion
export const parameters = { docs: { inlineStories: false } };
```

File: code/addons/docs/react/README.md [hunk 2]
```diff
@@ -98,14 +98,15 @@ Some **markdown** description, or whatever you want.
...
+
+To do so for all stories, update `.storybook/preview.js`:
 
 ```js
-export const parameters = {
-  docs: {
-    inlineStories: false,
-  },
-};
+export const parameters = { docs: { inlineStories: true } };
```
Comment (shilman) [inline code/addons/docs/react/README.md:109]:
```suggestion
export const parameters = { docs: { inlineStories: false } };
```

File: code/addons/docs/vue/README.md [hunk 3]
```diff
@@ -133,18 +133,14 @@ Yes, it's redundant to declare `component` twice. [Coming soon](https://github.c
...
-import { addParameters } from '@storybook/vue';
+To do so for all stories, update `.storybook/preview.js`:
 
-addParameters({
-  docs: {
-    inlineStories: true,
-  },
-});
+```js
+export const parameters = { docs: { inlineStories: true, }, };
```
Comment (shilman) [inline code/addons/docs/vue/README.md:143]:
```suggestion
export const parameters = { docs: { inlineStories: false, }, };
```

File: code/addons/docs/vue3/README.md [hunk 4]
```diff
@@ -133,18 +133,14 @@ Yes, it's redundant to declare `component` twice. [Coming soon](https://github.c
...
-import { addParameters } from '@storybook/vue';
+To do so for all stories, update `.storybook/preview.js`:
 
-addParameters({
-  docs: {
-    inlineStories: true,
-  },
-});
+```js
+export const parameters = { docs: { inlineStories: true } };
```
Comment (shilman) [inline code/addons/docs/vue3/README.md:143]:
```suggestion
export const parameters = { docs: { inlineStories: false } };
```

Comment (shilman) [reply code/addons/docs/vue3/README.md:143]: Add iframeHeight to example also?

File: code/addons/docs/web-components/README.md [hunk 5]
```diff
@@ -112,24 +112,15 @@ For a full example see the [web-components-kitchen-sink/custom-elements.json](..
...
-});
-```
+However, you can render stories in an iframe, with a default height of `60px` (configurable using the `docs.iframeHeight` story parameter), by using the `docs.inlineStories` parameter.
 
-or add it to individual stories.
+To do so for all stories, update `.storybook/preview.js`:
 
 ```js
-<Story inline={false} />
+export const parameters = { docs: { inlineStories: true } };
```
Comment (shilman) [inline code/addons/docs/web-components/README.md:123]:
```suggestion
export const parameters = { docs: { inlineStories: false } };
```

**Review by shilman — CHANGES_REQUESTED** [review]

**Review by tmeasday — COMMENTED** [review]
x

**Comment by tmeasday** [comment]
I decided not to add the height parameter in case people thought it was needed, but I can change it.

**Review by ndelangen — APPROVED** [review]

**Pull request merged**